demo_fastapi/sales.py

```diff
@@ -57,27 +73,118 @@ def create_order(
         )
 
     # if everything ok, make an order
-    order_id = uuid1().int
+    order_id = random.randrange(10**8)  # 12345  # uuid1().int
     order = Order(
         id=order_id, datestamp=datestamp, buyer=buyer, apples=apples, oranges=oranges
     )
+
+    # save to the database
+    cursor = cnxn.cursor()
+    try:
+        cursor.execute(
+            "INSERT INTO ShoppingList VALUES (?,?,?,?,?)",
+            datestamp,
+            buyer,
+            apples,
+            oranges,
+            order_id,
+        )
+        cursor.commit()
```

Comment:
could we test this functionality without the api as well?

Reply:
I am unsure what you mean? Let's talk about it :)